src/features/Builder/ActivityItems/ItemConfiguration/GroupedSelectSearchController/ItemTypeTooltip/TooltipComponents/Components/SelectionPerRow.tsx

```diff
@@ -0,0 +1,81 @@
+import { useTranslation } from 'react-i18next';
+import uniqueId from 'lodash.uniqueid';
+
+import { Svg } from 'components';
+import { StyledBodyMedium } from 'styles/styledComponents';
+
+import {
+  StyledPresentation,
+  StyledTooltipText,
+  StyledMatrixLine,
+  StyledMatrixLineElement,
+} from '../TooltipComponents.styles';
+import { SelectionProps, SelectionUiType } from '../TooltipComponents.types';
+
+export const SelectionPerRow = ({ uiType }: SelectionProps) => {
+  const { t } = useTranslation();
+  const isSingleSelection = uiType === SelectionUiType.Single;
+
+  const commonProps = {
+    width: '17',
+    height: '17',
+  };
+
+  const getSvgId = (rowIndex: number, colIndex: number) => {
```

Comment:
if you move isSingleSelection into function argument this fucntion also can be moved outside of component scope